package.json

```diff
@@ -8,10 +8,9 @@
     "install": "lerna bootstrap --hoist && npm run build",
     "lint": "npm run lint:code && npm run lint:text && npm run lint:types",
     "lint:code": "eslint --cache --cache-location node_modules/.cache/eslint --cache-strategy content --ext js,ts,tsx .",
-    "lint:text": "prettier --check '**/*.{json,md,mdx,yml}'",
+    "lint:text": "prettier --check **/*.{json,md,mdx,yml}",
```

Comment:
Me committing this change was a mistake so I'll revert this as it's not relevant to the PR.

I only modify this locally when working because else the command just simply doesn't work on Windows. I immediately get the error:
```
> lint:text
> prettier --check '**/*.{json,md,mdx,yml}'

Checking formatting...
[error] No files matching the pattern were found: "'**/*.{json,md,mdx,yml}'".
All matched files use Prettier code style!
```

Reply:
Interesting... Maybe it's something Prettier could improve?